### PR TITLE
[CSM-499] Expose list of update proposals

### PR DESCRIPTION
**Pull request by Pastafarianist**

### CODE REVIEW

File: lib/src/Pos/Wallet/Web/ClientTypes/Instances.hs [hunk 1]
```diff
@@ -157,12 +158,12 @@ instance FromMultipart CElectronCrashReport where
...
 instance HasTruncateLogPolicy CTx where
     -- Related logs are printed often.
-    -- Tx history changes increamentally, order is newest first
+    -- Tx history changes incrementally, order is newest first
     truncateLogPolicy = take 5
 
 instance HasTruncateLogPolicy CWallet where
     -- Related logs are printed seldom.
-    -- Contains constantly chaning info (balance)
+    -- Contains constantly changing info (balance)
```
Comment (Martoon-00) [inline lib/src/Pos/Wallet/Web/ClientTypes/Instances.hs]: I have a feeling that I typed these on
https://im0-tub-ru.yandex.net/i?id=7a4daadd3b98690d51f7ad5e56b581f8&n=13

**Comment by gromakovsky** [comment]
>  Discuss whether this is needed for cardano-sl-1.0

Please no 🙏

File: wallet/src/Pos/Wallet/Web/Methods/Misc.hs [hunk 2]
```diff
@@ -85,6 +89,18 @@ applyUpdate :: (MonadWalletDB ctx m, MonadUpdates m) => m ()
 applyUpdate = removeNextUpdate >> applyLastUpdate
 
 ----------------------------------------------------------------------------
+-- Update proposals
+----------------------------------------------------------------------------
+
+-- | Get info on all proposals for the specified app
```
Comment (gromakovsky) [inline wallet/src/Pos/Wallet/Web/Methods/Misc.hs]: Which app?

File: wallet/src/Pos/Wallet/Web/Swagger/Description.hs [hunk 3]
```diff
@@ -158,6 +158,11 @@ instance HasCustomSwagger ApplyUpdate where
         "Apply last update."
 
 
+instance HasCustomSwagger ConfirmedProposals where
+    swaggerModifier = modifyDescription
+        "Get info on all proposals for the specified app."
```
Comment (gromakovsky) [inline wallet/src/Pos/Wallet/Web/Swagger/Description.hs]: Which app?

**Pull request merged**